recipe/meta.yaml

```
@@ -31,16 +31,26 @@ requirements:
   run_constrained:
     - python !=3.6.0
```

https://github.com/domdfcoding/handy-archives/blob/v0.2.0/pyproject.toml#L10
```suggestion
    - python >=3.6.1
```